libs/design/src/atoms/image/image.module.ts

```diff
@@ -13,5 +13,8 @@ import { DaffImageComponent } from './image.component';
   exports: [
     DaffImageComponent,
   ],
+  entryComponents: [
+    DaffImageComponent,
```

Comment:
This should no longer be necessary with Ivy.

Reply:
aren't we still supporting pre-Angular 9 apps?

Reply:
Indeed.